RecoTracker/PixelSeeding/plugins/alpaka/CAHitNtupletGenerator.h

```diff
@@ -0,0 +1,84 @@
+#ifndef RecoPixelVertexing_PixelTriplets_Alpaka_CAHitNtupletGenerator_h
+#define RecoPixelVertexing_PixelTriplets_Alpaka_CAHitNtupletGenerator_h
+
+#include <alpaka/alpaka.hpp>
+
+#include "DataFormats/Track/interface/alpaka/TrackSoADevice.h"
+#include "DataFormats/Track/interface/TrackSoAHost.h"
```

Comment:
After the type aliases are set correctly in `TrackSoADevice.h`, this `#include` should not be needed
```suggestion
```